src/odemis/driver/test/xt_client_test.py

```diff
@@ -1132,18 +1131,21 @@ def test_multiprobe_mode_VA(self):
         current_aperture_index = self.scanner.apertureIndex.value
         current_beamlet_index = self.scanner.beamletIndex.value
 
+        # Test multiple time to see if the change from True --> False and the other way around succeeds also if it is
+        # preceded by switch in beam mode via the VA.
         for multi_beam_boolean in [True, False, True, False, True]:
             self.scanner.multiBeamMode.value = multi_beam_boolean
-            time.sleep(6)
+            time.sleep(8)
```

Comment:
Upon another check with Wilco he thought that waiting 8 seconds would be better.